app/helpers/view_components/student/tracks_list.rb

```diff
@@ -1,14 +1,18 @@
 module ViewComponents
   module Student
     class TracksList < ViewComponent
-      def initialize(data, request = default_request)
+      def initialize(user, data, request = default_request)
+        @user = user
         @data = data
         @request = request
       end
 
       def to_s
         react_component("student-tracks-list", {
-                          request: request.deep_merge({ options: { initialData: data } }),
+                          request: request.deep_merge({
+                                                        options: { initialData: data },
+                                                        query: { auth_token: user.auth_tokens.first.to_s }
```

Comment:
We now need to pass the auth token in order for the API to respond.

Reply:
Thanks. Yeah - this is temporary until the API supports sessions. I've added a TODO to remove it again in the future that in this PR.